### PR TITLE
Show registration facility and date in detail screen

### PR DESCRIPTION
**Pull request by Honey14**
https://app.clubhouse.io/simpledotorg/story/213/display-patient-s-registration-facility-in-the-patient-summary-screen

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt [hunk 1]
```diff
@@ -287,12 +288,29 @@ class PatientSummaryScreen(
...
 
+  private fun displayRegistrationFacilityName(patientSummaryProfile: PatientSummaryProfile) {
+    val registeredFacilityName = patientSummaryProfile.facilityName
+    val recordedAt = patientSummaryProfile.patient.recordedAt
+    val recordedDate = dateFormatter.format(recordedAt.toLocalDateAtZone(userClock.zone))
+
+    if (registeredFacilityName != null) {
+      facilityNameAndDateTextView.visibility = View.VISIBLE
+      labelRegistered.visibility = View.VISIBLE
+      val facilityNameAndDate = "$recordedDate at $registeredFacilityName"
```
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt:306]: Can we move this to string resources? This needs localisation.

**Comment by vinaysshenoy** [comment]
I tried running this on a device, and the visibility of the fields are not getting toggled. The patient record has the fields set but they don't show. Please take a look?

**Comment by msasikanth** [comment]
Picking up this PR since Honey is on leave.

**Comment by vinaysshenoy** [comment]
> I tried running this on a device, and the visibility of the fields are not getting toggled. The patient record has the fields set but they don't show. Please take a look?

@msasikanth We paired on this and found out the issue was related to the patient profile query not reading the registration and assigned facility IDs. We would need to update the query and relevant tests to verify this.

**Pull request merged**